integrations/apify/integration.definition.ts

```diff
@@ -0,0 +1,129 @@
+import { IntegrationDefinition, z } from '@botpress/sdk';
+import { integrationName } from './package.json';
+
+export default new IntegrationDefinition({
```

Comment:
praise: I am so happy to see this integration definition with everything written down here. It's so easy to read and understand what this does!